### ISSUE TITLE
FOV Is still broken

### ISSUE THREAD

**Issue by Stringer** [drdaxxy/gta5fov#20]
Hey man, I was just wondering if this mod would ever get a fix, it makes the first person in GTA V  ten times better but now it only works in vehicles.
Cheers,

**Comment by ItsMos** [comment]
yeah vehicle fov works but can i try this online?

**Comment by bobnassar93** [comment]
@ItsMos yes it's safe to use it online, i used to all the time before it was broken
